### PR TITLE
Update pt.json

### PR DESCRIPTION
**Pull request by leonardofelin**
Full review of Portuguese Language.

### CODE REVIEW

**Comment by leonardofelin** [comment]
Sorry, I don't know how to add the two files in the same pull request.

**Comment by felixlinker** [comment]
> Sorry, I don't know how to add the two files in the same pull request.

No worries! Happy to see any help :)

File: src/app/i18n/pt.json [hunk 1]
```diff
@@ -195,48 +238,54 @@
     "range": "Alcance",
     "ranget": "Alcance",
     "regen": "Taxa de regeneração",
-    "reload": "Recarregar",
-    "rof": "Taxa de rajada",
+    "reload": "Tempo de recargar,
```
Comment (felixlinker) [inline src/app/i18n/pt.json:241]: You missed a closing quote here.

**Review by felixlinker — CHANGES_REQUESTED** [review]
There are some issues with your changes. Do you know how to use git on your local computer? Otherwise I would go ahead and fix these.

File: src/app/i18n/pt.json (hunk 1 above)
Comment (leonardofelin) [reply src/app/i18n/pt.json:241]: I'll fix them right away.

Comment (leonardofelin) [reply src/app/i18n/pt.json]: I think I fixed it. I don't really know how to use git to be honest. I need to learn this better.
from "reload": "Tempo de recargar,
to "reload": "Tempo de recarga",

**Comment by felixlinker** [comment]
Very nice, thanks for addressing this so quickly!

**Pull request merged**